test/intl402/Locale/constructor-parse-twice.js

```diff
@@ -94,3 +52,13 @@ for (const {tag, options, canonical} of testData) {
     `new Intl.Locale("${tag}", ${options}).toString() returns "${canonical}"`
   );
 }
+
+assert.throws(RangeError, () =>
+    new Intl.Locale("no-bok", {region: "NO", calendar: "gregory"}).toString());
```

Comment:
these `.toString()` should never happen, right? Please remove them in the other files as well.